_content/tour/ind/constants.article

```diff
@@ -1,123 +1,121 @@
-Constants
-One of the more unique features of Go is how the language implements constants.
+Konstanta
+Salah satu fitur unik dari Go adalah bagaimana bahasa tersebut mengimplementasikan konstanta.
 
-* Constants
+* Konstanta
 
-- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Watch The Video]]
-- Need Financial Assistance, Use Our [[https://www.ardanlabs.com/scholarship/][Scholarship Form]]
+- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Tonton Video]]
+- Perlu bantuan finansial, Isi [[https://www.ardanlabs.com/scholarship/][Formulir Beasiswa]]
 
-One of the more unique features of Go is how the language implements constants.
-The rules for constants in the language specification are unique to Go. They provide
-the flexibility Go needs to make the code we write readable and intuitive while still
-maintaining type safety.
+Salah satu fitur unik dari Go adalah bagaimana bahasa tersebut mengimplementasikan konstanta. 
+Aturan untuk konstanta dalam spesifikasi bahasa ini unik untuk Go. Mereka memberikan 
+fleksibilitas yang dibutuhkan Go untuk membuat kode yang kita tulis mudah dibaca dan intuitif 
+sambil tetap menjaga keamanan tipe data.
 
-Constants can be typed or untyped. When a constant is untyped, it’s considered to
-be of a kind. Constants of a kind can be implicitly converted by the compiler. This
-all happens at compile time and not at runtime.
+Konstanta bisa memiliki jenis atau tidak. Ketika sebuah konstanta tidak memiliki jenis, 
+itu dianggap sebagai "jenis". Jenis konstanta dapat dikonversi secara implisit oleh kompiler. 
```

Comment:
please change `kompiler` to `kompilator`

Reply:
ack